arm/container-group.json

```diff
@@ -0,0 +1,91 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "groupIndex": {
+            "type": "int"
+        },
+        "containerCount": {
```

Comment:
Consider adding `"minValue: 1"` here.

Reply:
Validated on script side -> i don't like to idea duplicate those limits around. Check there is more sophisticated since it also calculates total allowed etc in group.